lib/image.js

```diff
@@ -46,20 +43,22 @@ Object.defineProperty(Image.prototype, 'src', {
             throw err
           }
         }
-
-        if (!get) get = require('simple-get')
-
-        get.concat({
-          url: val,
+        const request =  fetch(url = val, data = {
+          method: "POST",
```

Comment:
Can you use single quotes to align with the rest of the file?

```suggestion
          method: 'POST',
```

Reply:
=> https://github.com/Automattic/node-canvas/pull/2309/commits/93cfd2f012cf40bbf0d7a7f88f60b9c3659106a2